python/identifiers.cpp

```diff
@@ -36,7 +44,9 @@ void register_identifiers(pybind11::module& m) {
         .def("__str__", [](arb::cell_member_type m) {return pprintf("<arbor.cell_member: gid {}, index {}>", m.gid, m.index);})
         .def("__repr__",[](arb::cell_member_type m) {return pprintf("<arbor.cell_member: gid {}, index {}>", m.gid, m.index);});
 
-    pybind11::enum_<arb::cell_kind>(m, "cell_kind",
+    py::implicitly_convertible<py::tuple, arb::cell_member_type>();
```

Comment:
It's so we can write `(0, 1)` in arguments to functions that take a `cell_member`, rather than having to write `cell_member((0, 1))`. It cuts out a fair bit of verbosity.

Reply:
Okay, thanks :)